weaver/utils.py

```diff
@@ -431,17 +431,22 @@ def fetch_file(file_reference, file_outdir):
     :param file_outdir: Output directory path of the fetched file.
     :return: Path of the local copy of the fetched file.
     """
+    file_href = file_reference
     file_path = os.path.join(file_outdir, os.path.basename(file_reference))
     if file_reference.startswith("file://"):
         file_reference = file_reference[7:]
+    LOGGER.debug("Fetch file resolved:\n"
+                 "  Reference: [%s]\n"
+                 "  File Path: [%s]", file_href, file_path)
     if os.path.isfile(file_reference):
-        shutil.copyfile(file_reference, file_path)
+        shutil.copyfile(file_reference, file_path, follow_symlinks=False)
```

Comment:
Does symlinks caused issue? I'd like to have a comment justifying this change.

Reply:
It doesn't cause an issue in itself, but I found this option while looking through pywps and cwltool code. 
It basically avoids an hard-copy of the file by only copying the symlink if the referenced path is a symlink. PyWPS drops symlinks to avoid copying big files, so could save lots of time and space by not over duplicating existing files.